src/modules/75-cf/pages/feature-flags-detail/targeting-rules-tab/hooks/__tests__/useFeatureEnabled.test.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2023 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import { renderHook, RenderHookResult } from '@testing-library/react-hooks'
+import type { FC } from 'react'
+import React from 'react'
+import { TestWrapper } from '@common/utils/testUtils'
+import * as rbacHooksMock from '@rbac/hooks/usePermission'
+import useFeatureEnabled, { UseFeatureEnabledProps, UseFeatureEnabledReturn } from '../useFeatureEnabled'
+
+const renderHookUnderTest = (
+  props: Partial<UseFeatureEnabledProps> = {}
+): RenderHookResult<undefined, UseFeatureEnabledReturn> => {
+  const wrapper: FC = ({ children }) => {
```

Comment:
This will kick up a fuss with React 18 since `FC` won't include `children` any more. Might as well get ahead of it.
```suggestion
  const wrapper: FC<PropsWithChildren<unknown>> = ({ children }) => {
```